.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
`.eslintrc` is deprecated, they recommend `js`, `yml` or `json`. Gone with `json` just cos.